delocate/__init__.py

```diff
@@ -1,12 +1,20 @@
 # Init for delocate package
 
+import warnings
+
 from .delocating import delocate_path, delocate_wheel, patch_wheel
 from .libsana import tree_libs, wheel_libs
 
 try:
     from ._version import __version__
 except ImportError:
     __version__ = "0.0.0"
+    warnings.warn(
```

Comment:
But - what do you think about planning to raise an error after a period of deprecation?

Reply:
Your party, your rules!   Just go for whatever you prefer.